src/pages/Home.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import { useQuery } from 'react-query'
 import  Whoops404  from './Error';
+import { Link } from 'react-router-dom';
 
 const fetchProducts = async () => {
-  const res = await fetch(`https://dummyjson.com/products`)
-    return res.json()   
-}
-  
+  const response = await (await fetch(`https://dummyjson.com/products`)).json();
```

Comment:
the whole issues came about because the initial way was returning a promise so i ended up adding another `await`